src/scripts/OSFramework/OSUI/Helper/Device.ts

```diff
@@ -460,6 +460,21 @@ namespace OSFramework.OSUI.Helper {
 			return DeviceInfo._isTouch;
 		}
 
+		/**
+		 * Method to return if is a mobile device based on is operating system
```

Comment:
comment seems confusing

Reply:
@BenOsodrac I can improve the comment, however this method is to check if we are in a mobile device. The validation are based on device operating system, that will returns `ios` or `android` in nowadays.